jekyll/_cci2/insights-partnerships.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: classic-docs
+title: "Insights Partnerships"
+short-title: "Insights Partnerships"
+version:
+- Cloud
+---
+
+## Overview
+
+{:toc}
+
+This document describes how you can connect Insights data with third party
+providers.
+
+### Sumo Logic integration
+
+Sumo Logic users may track and visualize analytical data across all of their
+jobs on CircleCI. To do so, use the Sumo Logic Orb and Sumo Logic app
+integration from the Sumo Logic partner integrations site.
+
+
+#### The CircleCI dashboard for Sumo Logic
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 16. You can fix by inserting `{: #the-circleci-dashboard-for-sumo-logic }` to the next line